src/utils/structs/checkpoints.cairo

```diff
@@ -159,17 +159,20 @@ impl CheckpointImpl of CheckpointTrait {
     }
 }
 
+const _2_pow_184: felt252 = 0x10000000000000000000000000000000000000000000000;
+
 /// Packs a Checkpoint into a (felt252, felt252).
 ///
 /// The packing is done as follows:
 /// - The first felt of the tuple contains `key` and `value.low`.
-/// - `key` is stored at the 64 most significant bits.
-/// - `value.low` is stored at the 128 less significant bits.
+/// - `key` is stored at range [4,67] bits (0-indexed), taking most significant bits first.
+/// - `value.low` is stored at the 128 less significant bits (at the end).
```

Comment:
```suggestion
/// - The first four bits are skipped to avoid representation errors due to `felt252` being a bit less than 252 full bits
/// (https://docs.starknet.io/documentation/architecture_and_concepts/Cryptography/p-value/)
/// - `key` is stored at range [4,67] bits (0-indexed), taking the most significant usable bits.
/// - `value.low` is stored at range [124, 251], taking the less significant bits (at the end).
```
i think showing ranges for both values is a bit more consistent and easy to follow